app/server.ts

```diff
@@ -8,7 +9,8 @@ const logger: Logger = new Logger();
 const logLabel: string = 'server.ts';
 
 app.listen(port, () => {
-  logger.trace(`Server  listening on port ${port}`, logLabel);
+  S2SService.getInstance();
```

Comment:
maybe would be worth it to add the instance to app.locals so is accessible in every controller without having to call getInstance()?